opensrp-chw/src/nacp/assets/json.form/child_hv_pmtct.json

```diff
@@ -179,7 +179,7 @@
         "openmrs_entity": "concept",
         "openmrs_entity_id": "",
         "type": "toaster_notes",
-        "text": "CHW Information: Advice the mother to go for the next checkup when the child is 18 months",
+        "text": "CHW Information: Advice the mother to go for the next HIV testing:\n3 months after giving birth,\n6 months after giving birth and,\n18 months after giving birth",
```

Comment:
@devcolonel - Here if you want a newline in toaster message you need to use <br > instead if \n. Do you mind confirming if \n will work?